src/@types/mongo/User.ts

```diff
@@ -10,6 +10,7 @@ export type TemporaryIdentity = {
   eventId: string;
   value: string;
   type: string;
+  ttl: number;
```

Comment:
TTL is supposed to be a duration, how do you know when it has expired?

Wouldn't it be simpler with `expiresAt`?

Reply:
`expiresAt` should make the code more readable.